index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile First Porfolio Website</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <!--MAIN CONTENT-->
+    <main class="main">
+
+        <!--Header-->
+        <header>
```

Comment:
Great design @borrnelsonk 🤝

- [ ] As the previous reviewer pointed out, kindly make sure your design is a close replica of the design provided in the Figma template. You may please fix the font, spacing, colors, etc.
- Your design
![your design](https://user-images.githubusercontent.com/63917725/186647597-b87876eb-3608-4a33-8df2-4b6f8d5a6872.PNG)

- Figma design
![Figma design](https://user-images.githubusercontent.com/63917725/186647632-b98bafe1-d54f-437c-ba77-1e36d78a1fbd.PNG)

Reply:
- [ ] As the previous reviewers pointed out, your design is a bit far from your chosen Figma template. Kindly make sure your design is a close replica of the design provided in the Figma template. This includes, font, font-size, font-family, color, font-weight, margin, padding among others.

### Your Design

![your-toobar-design](https://user-images.githubusercontent.com/26597432/186681045-c131eb29-2275-4f44-9d39-d5144210d42f.png)


#### Figma Design

![figma-toolbar-design](https://user-images.githubusercontent.com/26597432/186681225-8edfdcde-388f-4618-a62f-152bc2ef43fb.png)


### Your Design

![your-headline-design](https://user-images.githubusercontent.com/26597432/186681364-cb49ad6d-e389-4c69-a79d-98eb83d43cb1.png)


### Figma Design

![figma-headline-design](https://user-images.githubusercontent.com/26597432/186681488-ae53344d-b9ff-4f99-b54f-9ebf4a110778.png)